master/internal/api_user_intg_test.go

```diff
@@ -924,3 +926,207 @@ func getActivityEntry(ctx context.Context, userID model.UserID, entityID int32)
 	return db.Bun().NewSelect().Model((*model.UserActivity)(nil)).Where("user_id = ?",
 		int32(userID)).Where("entity_id = ?", entityID).Count(ctx)
 }
+
+func TestPostLongLivedToken(t *testing.T) {
+	api, curUser, ctx := setupAPITest(t, nil)
+
+	// Without lifespan input
+	token, err := api.PostLongLivedToken(ctx, &apiv1.PostLongLivedTokenRequest{})
+	require.NoError(t, err)
+	require.NotNil(t, token)
+
+	exists, err := getLongLivedTokensEntry(ctx, curUser.ID)
+	require.True(t, exists)
+	require.NoError(t, err)
```

Comment:
nit (non-blocking): It might also be a solid move to include some cleanup steps to these tests, we don't necessarily want a bunch of test access tokens sticking around in the DB for test users